ajax/libs/dollar.js/package.json

```diff
@@ -23,5 +23,14 @@
     "AMD",
     "oz",
     "ozjs"
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dexteryy/DollarJS.git",
+    "basePath": "",
+    "files": [
+      "dollar*.js",
+      "dollar/**/*"
```

Comment:
Why do we need? I think this auto-update is not good at all, both line 32 and line 33.